test/source-updater.test.js

```diff
@@ -810,48 +810,50 @@ QUnit.test('setDuration waits for video buffer to finish updating', function(ass
   assert.ok(this.sourceUpdater.updating(), 'updating during appends');
 });
 
-QUnit.test(
-  'setDuration waits for both audio and video buffers to finish updating',
-  function(assert) {
-    const done = assert.async();
-    let appendsFinished = 0;
+if (!videojs.browser.IS_EDGE) {
```

Comment:
This test is failing intermittently on edge, even in master, since edge throws on setDuration sometimes.

Reply:
Same comment about skipping here.